README.md

```diff
@@ -102,9 +102,22 @@ NOTE: OpenShift cluster consumes a lot of resources, you should have at least 18
 
 You should run `gocli` command:
 ```bash
-gocli run okd --prefix okd-4.1.0-rc.0 --background kubevirtci/okd-4.1.0-rc.0@sha256:71afc34d9299aa11313b43a584e7e9d7e2f962279453b53d853a9d3bcb8b3255
+gocli run okd --prefix okd-4.1.0 --background kubevirtci/okd-4.1.0@sha256:9e53873338f5f0ca97273e64c16717e53c3b36fc1f971c4dfade9df4f895cea2
 ```
 
+### How to connect to the OKD console
+
+To connect the OKD console you should add once hosts to the `/etc/hosts`
+
+```bash
+127.0.0.1 console-openshift-console.apps.test-1.tt.testing
+127.0.0.1 oauth-openshift.apps.test-1.tt.testing
+```
+
+and specify the `--ocp-console-port` under the `gocli` run command to `443`.
```

Comment:
maybe add it to the run command above? It's a bit of a pity when you execute the command and then read afterwards that you need more ;)

Reply:
hh true, will add it

Reply:
done